core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -723,32 +557,25 @@ class UserService(
             )
             Seq(getV2BillingPolicy(projectAccessUpdate.role))
         }
-        _ <- addUserToBillingProjectInner(projectName, projectAccessUpdate, policies)
+        _ <- Future.traverse(policies) { policy =>
+          samDAO
+            .addUserToPolicy(
+              SamResourceTypeNames.billingProject,
+              projectName.value,
+              policy,
+              projectAccessUpdate.email,
+              ctx
+            )
+            .recoverWith { case regrets: Throwable if policy == SamBillingProjectPolicyNames.canComputeUser =>
```

Comment:
Now that I look at it, this recover block doesn't look like a great idea in the first place. If it fails, it still returns success. It logs the error, but you'd have to know to look for it.
I guess the main thing it would do, is make sure the user is added to any remaining policies in the list.

Reply:
The idea was to not throw on failure because v2 billing projects don't have this policy. We still don't want to throw, but I think we can skip adding users to the `canCompute` billing project policy entirely now. We have [WOR-658](https://broadworkbench.atlassian.net/browse/WOR-658) to clean up the policies on older billing projects